frontend/src/views/WheresMyGene/components/InfoPanel/components/Legend/index.tsx

```diff
@@ -67,7 +67,7 @@ export default memo(function Legend({
             allChartProps={allChartProps}
             availableFilters={availableFilters}
             tissues={tissues || EMPTY_ARRAY}
-            expandedTissues={expandedTissues || EMPTY_ARRAY}
+            expandedTissues={expandedTissues ?? new Set()}
```

Comment:
new Set() will fail `===` check, maybe create a new const `EMPTY_SET` in `frontend/src/common/constants/utils.ts` and use it here?